inventory/sample.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -53,6 +91,11 @@ dns_nameservers:
 #master_root_volume: "/dev/sda1"
 #master_root_volume_size: 50
 
+# etcd(s) root (/) volume size and device
```

Comment:
All of these definitions for root volumes/size, etc. should be part of the `cloud_infrastructure` above

Reply:
NOTE: remove `*_root_volume` variable -- not required

Reply:
Done